files/manifests/k8s-api-server.yaml.tmpl

```diff
@@ -38,8 +38,8 @@ spec:
     - --profiling=false
     - --service-account-lookup=true
     - --authorization-mode=RBAC
-    - --feature-gates=TTLAfterFinished=true{{ if .EnableCronJobTimeZone }},CronJobTimeZone=true{{ end }}
-    - --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,DefaultStorageClass,PersistentVolumeClaimResize,PodSecurityPolicy,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook
+    {{ if .EnableCronJobTimeZone }}- --feature-gates=CronJobTimeZone=true{{ end }}
```

Comment:
I think we don't need to add it anymore, it's by default in 1.25
<img width="795" alt="image" src="https://github.com/giantswarm/k8scloudconfig/assets/1936982/1f3dee49-68d1-4053-a35b-ff2a7428cf46">

Reply:
good catch, but this and other vintage repos will be a thing of the past soon so I guess I'll ignore it